rt-forecast/functions/format-forecast.R

```diff
@@ -49,6 +47,9 @@ format_forecast<- function(forecasts,
   # drop unnecessary columns
   forecasts_format <- forecasts_format[, !c("epiweek", "region")]
   forecasts_format <- forecasts_format[target_end_date > forecast_date]
+  forecasts_format <- forecasts_format[, horizon := 1 + as.numeric(target_end_date - min(target_end_date)) / 7]
```

Comment:
couldn't you just do this without the assignment operator? `forecasts_format[, horizon := 1 + as.numeric(target_end_date - min(target_end_date)) / 7]`

Reply:
yes, I like it as a visual reminder of what is happening.